src/scss/index 2.scss

```diff
@@ -0,0 +1,2 @@
+@import "variables";
```

Comment:
Are you trying to customize the bootstrap with your variables? Where is the variables file?